packages/camera/index.android.ts

```diff
@@ -168,8 +168,16 @@ export let isAvailable = function () {
 	return Utils.android.getApplicationContext().getPackageManager().hasSystemFeature(android.content.pm.PackageManager.FEATURE_CAMERA);
 };
 
+function api33(): boolean {
+	return (<any>android).os.Build.VERSION.SDK_INT >= 33 && Utils.ad.getApplicationContext().getApplicationInfo().targetSdkVersion >= 33;
```

Comment:
Perhaps, we should make use of `SDK_VERSION` constant that was added in 8.4. We use it everywhere inside core now.

Reply:
I was hoping to make it a non breaking change, rather than force a core bump though.